DESCRIPTION

```diff
@@ -28,7 +28,31 @@ Description:
     https://covid19forecasthub.org/.
 License: MIT
 Imports:
-  dplyr (>= 0.8.4), tibble (>= 2.1.3), tidyr (>= 1.0.2), MMWRweek (>= 0.1.1), lubridate (>= 1.7.4), purrr (>= 0.3.3), scoringutils (>= 0.1.7), ggplot2, magrittr, forcats, RcppRoll, colorspace, ggnewscale, zoltr (>= 0.6.5), grDevices, RColorBrewer
+    dplyr (>= 0.8.4),
+    tibble (>= 2.1.3),
+    tidyr (>= 1.0.2),
+    MMWRweek (>= 0.1.1),
+    lubridate (>= 1.7.4),
+    purrr (>= 0.3.3),
+    scoringutils (>= 0.1.7),
+    ggplot2,
+    magrittr,
+    forcats,
+    RcppRoll,
+    colorspace,
+    ggnewscale,
+    zoltr (>= 0.6.5),
+    grDevices,
+    RColorBrewer,
+    stats,
+    utils,
+    readr,
+    httr,
+    stringr,
+    yaml,
+    covidData
```

Comment:
r.e. this dependency on covidData -- did you notice functions in the package that needed this other than `preprocess_jhu`?  Our hope is that a typical user of covidHubUtils would be able to use the package without having covidData installed; I think most other functions don't rely on its presence.  Did you notice other functions that relied on covidData, or were you getting errors if covidData was not listed in the imports?

`preprocess_jhu` is only used by us to output truth files to the hub repo, and we only put it into this package to give it a place where it could be formally and routinely unit tested. But if its presence is causing problems, my preference would be to move it somewhere else or figure out the right settings to make covidData optional-but-not-required.  And if there are other functions depending on covidData here, similar comments apply.

Reply:
There is also `preprocess_hospitalization` that uses it. 

I think I got a warning that funcitons were called using `covidData::` without it being an import - that's why I added it to imports.